sparkly/writer.py

```diff
@@ -255,6 +275,188 @@ def write_partition_to_kafka(messages):
 
         rdd.mapPartitions(write_partition_to_kafka).count()
 
+    def redis(self,
```

Comment:
@mantzouratos Redis supports different data structures. You picked only one of them to solve your use-case, but it doesn't mean that everybody will use it like that.

In addition to that, very specific decisions are:
- `keys` generation
- compression
- grouping into arrays